docs/website/content/docs/v0.3/Configuration/environments.md

```diff
@@ -27,7 +27,7 @@ metadata:
   name: default
 spec:
   kernel:
-    url: "https://github.com/talos-systems/talos/releases/download/v0.8.1/vmlinuz-amd64"
+    url: "https://github.com/talos-systems/talos/releases/download/v0.10.2/vmlinuz-amd64"
```

Comment:
I bumped those to 0.9.3 in my PR because that's what we use for testing

Reply:
let me bump our version to 0.10.2 then, I'll do that in the separate PR

we need Sidero to keep up to date with Talos!